src/configure/configure.ts

```diff
@@ -159,24 +160,40 @@ class Orchestrator {
         }
     }
 
+    private async setPipelineType() {
+        if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github && extensionVariables.enableGitHubWorkflow) {
+            this.pipelineType = PipelineType.GitHubPipeline;
+        } else {
+            this.pipelineType = PipelineType.AzurePipeline;
+        }
+    }
+
+    private isResourceAlreadySelected(): boolean {
+        return this.context['isResourceAlreadySelected'];
+    }
+
     private async getInputs(node: any): Promise<void> {
         telemetryHelper.setTelemetry(TelemetryKeys.FF_UseGithubForCreatingNewRepository,
             vscode.workspace.getConfiguration().get('deployToAzure.UseGithubForCreatingNewRepository'));
         telemetryHelper.setTelemetry(TelemetryKeys.FF_UseAzurePipelinesForGithub,
             vscode.workspace.getConfiguration().get('deployToAzure.UseAzurePipelinesForGithub'));
 
-        const resourceNode = await this.analyzeNode(node);
+        await this.analyzeNode(node);
 
         if (this.continueOrchestration) {
             await this.getSourceRepositoryDetails();
             if (!this.inputs.azureSession) {
                 this.inputs.azureSession = getAzureSession();
             }
             const repoAnalysisResult = await this.getRepositoryAnalysis();
-            await this.getSelectedPipeline(repoAnalysisResult);
-
+            await this.setPipelineType();
+            // Right click on not resource not supported for Azure pipeline
+            if (this.isResourceAlreadySelected() && this.pipelineType === PipelineType.AzurePipeline) {
+                throw Error("Scenario not supported for Azure pipelines.");
```

Comment:
I think you asked Pulkit for an error message. Is this final?

Reply:
Still waiting for his approval. Will merge only after I get that.

Reply:
@kanika1894  Lets merge this PR so that you can do bugbash tomorrow. the string can anyways be changed easily later.